elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -91,7 +91,7 @@
     private final DataStoreTransaction transaction;
     private final RequestScope requestScope;
     private int hashCode = 0;
-
+    public static final Set<String> ALL_FIELDS = null;
```

Comment:
While you are updating the code, can you add the `<T>` argument to type definition above?
```java
private final Type<T> type;
```

Reply:
Makes sense - but I think that should be a separate PR.  Small PRs are easier to fit in smaller time blocks for review.